hive3/src/main/java/org/apache/hadoop/hive/ql/io/orc/OrcSplit.java

```diff
@@ -191,12 +191,15 @@ public boolean hasFooter() {
   }
 
   /**
-   * @return {@code true} if file schema doesn't have Acid metadata columns
-   * Such file may be in a delta_x_y/ or base_x due to being added via
-   * "load data" command.  It could be at partition|table root due to table having
-   * been converted from non-acid to acid table.  It could even be something like
+   * A check for whether or not this file was originally created in a Hive ACID table.
+   *
+   * A file may be in a delta_x_y/ or base_x due to being added via
+   * "load data" command.  It could be located in a partition|table root due to table having
+   * been converted from non-acid to acid table.  It could also even have paths containing
    * "warehouse/t/HIVE_UNION_SUBDIR_15/000000_0" if it was written by an
    * "insert into t select ... from A union all select ... from B"
+   *
+   * @return true if the file was written originally by a Hive ACID table
```

Comment:
There is no need to change this `@return`. The new version is more confusing because it is up to the Javadoc reader to know what it means to be originally created in a Hive ACID table. Instead, just use the original, "true if the file schema does not have ACID metadata columns". That should also be the first sentence description.

In addition, Javadoc paragraphs should be separated by `<p>` on its own line.

Reply:
thanks, fixed